frontend/app/src/components/history/events/HistoryEventsList.vue

```diff
@@ -125,14 +126,33 @@ const [DefineTable, ReuseTable] = createReusableTemplate();
               :block-number="blockEvent?.blockNumber"
               class="break-words leading-6 md:col-span-3 xl:col-span-4"
             />
-            <RowActions
-              class="md:col-span-1"
-              align="end"
-              :delete-tooltip="t('transactions.events.actions.delete')"
-              :edit-tooltip="t('transactions.events.actions.edit')"
-              @edit-click="editEvent(item)"
-              @delete-click="deleteEvent(item)"
-            />
+            <div class="flex items-center justify-end">
+              <RowActions
+                class="md:col-span-1"
+                align="end"
+                :delete-tooltip="t('transactions.events.actions.delete')"
+                :edit-tooltip="t('transactions.events.actions.edit')"
+                @edit-click="editEvent(item)"
+                @delete-click="deleteEvent(item)"
+              />
+              <RuiTooltip
+                v-if="item.missingAccountingRule"
+                :popper="{ placement: 'top', offsetDistance: 0 }"
+                :open-delay="400"
+              >
+                <template #activator>
+                  <RuiButton
+                    variant="text"
+                    color="warning"
+                    icon
+                    @click="emit('show:missing-rule-action', item)"
+                  >
+                    <RuiIcon size="16" name="information-line" />
+                  </RuiButton>
+                </template>
+                {{ t('actions.history_events.missing_rule.title') }}
+              </RuiTooltip>
```

Comment:
I think we can put that to `RowActions`' default slot and avoid the wrapper div.

Reply:
oh okay, cool, will do...